hercules/python_simulators/solar_pysam.py

```diff
@@ -82,12 +100,11 @@ def step(self, inputs):
                 system_model.value(k, v)
             except Exception:
                 print(k)
-        #### TODO: Check with Brooke about this "except KeyError" line ####
-                # Brooke: I got errors with KeyError so changed it to Exception and it's working
-        # print('model params = ',self.model_params)
 
-        print("sim_time_s = ", inputs["py_sims"]["inputs"]["sim_time_s"])
-        sim_timestep = int(inputs["py_sims"]["inputs"]["sim_time_s"] / self.dt)
+        # sim_time_s = inputs["py_sims"]["inputs"]["sim_time_s"]
+        sim_time_s = inputs["time"]
```

Comment:
This change (from `inputs["py_sims"]["inputs"]["sim_time_s"]` to `inputs["time"]` is causing the `test_step` test in tests/solar_pysam_test.py to fail. Possibly, the `step_inputs` dictionary in tests/solar_pysam_test.py just need to be updated to add the "time" field?

Reply:
Fixed! Thanks for pointing this out